client/me/apps/style.scss

```diff
@@ -0,0 +1,35 @@
+.apps .apps__title {
```

Comment:
It shouldn't need the extra level of specificity, just `.apps__title` would be fine.